src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -103,7 +103,6 @@ public void loginUser(
           public void onSuccess(LoginResponse response) {
             storeAuthenticationReceipt(response.getAuthenticationReceipt());
             setNewAccessToken(response.getAccessToken(), callback);
-            clearQueryClientCache();
```

Comment:
`setNewAccessToken` calls `initializeFromExistingAccessTokenCookie`, so eventually the cache will be cleared, making this call redundant.